man/systemd.exec.xml

```diff
@@ -2822,406 +2822,423 @@ StandardInputData=SWNrIHNpdHplIGRhIHVuJyBlc3NlIEtsb3BzLAp1ZmYgZWVtYWwga2xvcHAncy
-              </row>
-            </tbody>
-          </tgroup>
-        </table>
-
-        </listitem>
-      </varlistentry>
-
-      <varlistentry>
-        <term><varname>$PIDFILE</varname></term>
-
-        <listitem><para>The path to the configured PID file, in case the process is forked off on behalf of a
-        service that uses the <varname>PIDFile=</varname> setting, see
-        <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-        for details. Service code may use this environment variable to automatically generate a PID file at
-        the location configured in the unit file. This field is set to an absolute path in the file
-        system.</para></listitem>
-      </varlistentry>
-
-    </variablelist>
+    <para>The general philosophy is to expose a small curated list of environment variables to processes.
+    Services started by the system manager (PID 1) will be started, without additional service-specific
+    configuration, with just a few environment variables. The user manager inherits environment variables as
+    any other system service, but in addition may receive additional environment variables from PAM, and,
+    typically, additional imported variables when the user starts a graphical session. It is recommended to
+    keep the environment blocks in both the system and user managers managers lean.</para>
+
+    <para>Hint: <command>systemd-run -P env</command> and <command>systemd-run --user -P env</command> print
+    the effective system and user service environment blocks.</para>
+
+    <refsect2>
+      <title>Environment variables set or propagated by the service manager</title>
+
+      <para>The following environment variables are propagated by the service manager or generated internally
+      for each invoked process:</para>
+
+      <variablelist class='environment-variables'>
+        <varlistentry>
+          <term><varname>$PATH</varname></term>
+
+          <listitem><para>Colon-separated list of directories to use when launching
+          executables. <command>systemd</command> uses a fixed value of
+          <literal><filename>/usr/local/sbin</filename>:<filename>/usr/local/bin</filename>:<filename>/usr/sbin</filename>:<filename>/usr/bin</filename></literal>
+          in the system manager. When compiled for systems with "unmerged <filename>/usr/</filename>"
+          (<filename>/bin</filename> is not a symlink to <filename>/usr/bin</filename>),
+          <literal>:<filename>/sbin</filename>:<filename>/bin</filename></literal> is appended. In case of
+          the the user manager, a different path may be configured by the distribution. It is recommended to
+          not rely on the order of entries, and have only one program with a given name in
```

Comment:
i'd drop references to legacy systems with unmerged /usr... iirc even debian switched now? and if not, we should stop supporting that in the docs, even if we carry on support for this in code for a bit longer

Reply:
I'm just moving that text around. I'd leave it for now... Hopefully we can drop the support for split-usr from the codebase in some not-too-distant feature. Let's remove other mentions at that time too.

Reply:
FYI, Debian's default is merged-usr for new installations, but old installations are not forcefully converted, and require an opt-in manual step or two. This will not change for Debian 11. Discussion undergoing for Debian 12 (so 2+ years from now), the proposal being dropping split-usr finally, but nothing decided yet.